ext/dcos-installer/dcos_installer/installer_analytics.py

```diff
@@ -0,0 +1,34 @@
+import analytics
+import os
+import uuid
+
+
+class InstallerAnalytics():
+    def __init__(self, config):
+        self.uuid = self.new_uuid()
+        self.bootstrap_version = os.getenv("BOOTSTRAP_ID", "")
```

Comment:
I'm going to change that in the train branch. Makes sense, there is no way this ever needs to runs without one